content/sensu-go/6.3/files/agent.yml

```diff
@@ -1,55 +1,63 @@
 ---
 # Sensu agent configuration
 
+
 ##
 # agent overview
 ##
 #name: "hostname"
 #namespace: "default"
-#subscriptions:
-#  - example
-#labels:
-#  example_key: "example value"
-#annotations:
-#  example/key: "example value"
 
-##
-# agent configuration
-##
-#backend-url:
-#  - "ws://127.0.0.1:8081"
-#cache-dir: "/var/cache/sensu/sensu-agent"
-#config-file: "/etc/sensu/agent.yml"
-#disable-assets: false
-log-level: "debug" # available log levels: panic, fatal, error, warn, info, debug
 
 ##
-# api configuration
+# often the same among agents
 ##
+#agent-managed-entity: false
 #api-host: "127.0.0.1"
 #api-port: 3031
+#assets-burst-limit: 100
+#assets-rate-limit: 1.39
+#backend-handshake-timeout: 15
+#backend-heartbeat-interval: 30
+#backend-heartbeat-timeout: 45
+#cache-dir: "/var/cache/sensu/sensu-agent"
+#config-file: "/etc/sensu/agent.yml"
+log-level: "debug" #available log levels: panic, fatal, error, warn, info, debug
+#detect-cloud-provider: false
 #disable-api: false
 #events-burst-limit: 10
 #events-rate-limit: 10.0
+#require-fips: false
+#require-openssl: false
+#socket-host: "127.0.0.1"
+#socket-port: 3030
+#disable-sockets: false
+#statsd-disable: false
+#statsd-event-handlers:
+#  - example_handler
+#statsd-flush-interval: 10
+#statsd-metrics-host: "127.0.0.1"
+#statsd-metrics-port: 8125
 
-##
-# authentication configuration
-##
-#user: "agent"
-#password: "P@ssw0rd!"
 
 ##
-# monitoring configuration
+# often customized
 ##
+#allow-list: /etc/sensu/check-allow-list.yaml
+#backend-url:
+#  - "ws://0.0.0.0:8081"
+#  - "ws://0.0.0.0:8082"
```

Comment:
This isn't a valid URL, perhaps use `ws://example.com:8081`.

`0.0.0.0` is only a valid address as a way to tell servers to bind to all addresses in a system. Clients, like sensu-agent here, won't understand it.

Additionally, it might be helpful to include a URL like `wss://example.com:8081`, to point out that a secure websocket connection has a different scheme than a plaintext one.